RecoMET/METAlgorithms/src/CSCHaloAlgo.cc

```diff
@@ -35,29 +39,43 @@ CSCHaloAlgo::CSCHaloAlgo()
   matching_dwire_threshold = 5.;
 
 
-  et_thresh_rh_hbhe=25; //GeV
+  et_thresh_rh_hbhe=25;//to be removed
+  dphi_thresh_segvsrh_hbhe=0.05;
+  dr_lowthresh_segvsrh_hbhe=-100;
+  dr_highthresh_segvsrh_hbhe=20;
+  dt_highthresh_segvsrh_hbhe=30;
```

Comment:
are these parameters set from some first principles and will never change?
If some need for a change is possible/planned, the parameters should be derived from configuration.